topics/transcriptomics/tutorials/mirna-target-finder/tutorial.md

```diff
@@ -0,0 +1,726 @@
+>
+>    {% snippet snippets/rename_dataset.md %}
+>
+{: .hands_on}
+
+
+# mRNA diffential expression analysis
+
+Once we have imported the data, we can begin to study how brassinosteroid exposure alters gene expression patterns.
+
+## Quality assessment of mRNA reads
+
+Due to technical limitation, sequencing is considered a error-prone process. In Illumina sequening platforms, substitution type miscalls are the dominant source of errors, which can lead to inconsistent results. Another factor that can interfere with our analyses is the presence of adapter contaminants, which can result in increased number of unaligned reads, since the adapter sequences are synthetic and do not occur in the genomic sequence.
+
+Sequence quality control is therefore an essential first step in your analysis. We will use two popular tools for evaluating the quality of our raw reads: __FastQC__ and __MultiQC__. 
+
+> ### {% icon comment %} Comments
+> In order to visualise the data from both collections together in the __MultiQC__ tool, it will be necessary to combine the results generated by __FastQC__.
+> For more information on the topic of quality control, please see our training materials [here](https://training.galaxyproject.org/training-material/topics/sequence-analysis/tutorials/quality-control/tutorial.html).
+{: .comment}
+
+> ### {% icon hands_on %} Hands-on: Initial quality check
+>
+> 1. {% tool [FastQC](toolshed.g2.bx.psu.edu/repos/devteam/fastqc/fastqc/0.72+galaxy1) %} with the following parameters:
+>    - {% icon param-collection %} *"Dataset collection"*: `Control mRNA`
+> 2. Rename the outputs as `FastQC unprocessed control mRNA: RawData` and `FastQC unprocessed control mRNA: Webpage`
+> 3. Repeat the previous step with the `BR treated mRNA` collection.
+> 4. {% tool [Merge Collections](__MERGE_COLLECTION__) %} with the following parameters:
+>    - In *"Input collections"*:
+>        - *"1.Input Collections"*: `FastQC unprocessed control mRNA: RawData`
+>        - *"2.Input Collections"*: `FastQC unprocessed BR treated mRNA: RawData`
+> 5. {% tool [MultiQC](toolshed.g2.bx.psu.edu/repos/iuc/multiqc/multiqc/1.8+galaxy1) %} with the following parameters:
+>    - In *"Results"*:
+>      - *"Which tool was used generate logs?"*: `FastQC`
+>      - {% icon param-collection %} *"Dataset collection"*: select the output generated in the previous step.
+>   - In *"Report title"*: `mRNA initial quality check`
+> 4. Click on the {% icon galaxy-eye %} (eye) icon and inspect the generated HTML file
+>
+{: .hands_on}
+
+> ### {% icon question %} Questions
+>
+> Is there any parameters that indicate the need to process the samples in order to improve its quality?
+>
+> > ### {% icon solution %} Solution
+> >
+> > All parameters are within accepted limits. However, the adapter content shows the presence of universal adapters in our readings, which can be removed to avoid possible interferences at later stages.
```

Comment:
```suggestion
> > All parameters are within acceptable limits. However, the adapter content shows the presence of universal adapters in our readings, which can be removed to avoid possible interferences at later stages.
```